ee/src/main/java/io/supertokens/ee/EEFeatureFlag.java

```diff
@@ -151,6 +151,26 @@ public void syncFeatureFlagWithLicenseKey()
             licenseKey = this.getLicenseKeyFromDb();
             this.isLicenseKeyPresent = true;
         } catch (NoLicenseKeyFoundException ex) {
```

Comment:
- missing tests
- This is also in the wrong place. You need to do this as part of the `getPaidFeatureStats` function since the API call to //ee/featureflag also needs to reflect the multi tenancy stuff.
- The current app may have a license key which has no multi tenancy. So you need to also check the root key in that case if it has multi tenancy or not.

Reply:
added test, other points were already handled.